content/en/real_user_monitoring/session_replay/browser/troubleshooting.md

```diff
@@ -25,7 +25,8 @@ If you experience unexpected behavior with Datadog Session Replay, use this page
 
 Session Replay does not support:
 
-- The following HTML elements: `iframe`, `video`, `audio`, or `canvas`
+- The following HTML elements: `iframe without srcDoc`, `video`, `audio`, or `canvas`
```

Comment:
It should be different FMU. For <iframe srcDoc="" />, I think we [mask the attribute completely](https://github.com/DataDog/browser-sdk/blob/0764e799be92f32d5752cfb29d4f6f5220c42f12/packages/rum/src/domain/record/serialization/serializeAttribute.ts#L73-L76) if opt-in. And if it is not masked, I think we do capture the srcDoc or part of it depends on the content, because they are attributes.

Reply:
Ah right, we record the initial document but not the interaction/mutations within the document.

The same thing kind of applies with `src=xxx`. We record `src` attribute changes, so in the replay we will display the embedded website (like an image, we'll load the original URL). But we don't record iteraction/mutations inside of it.